### ISSUE TITLE
[RFC] Show Contenttype Database alterations through new options in nut CLI

### ISSUE THREAD

**Issue by brewneaux** [bolt/bolt#5595]
I'm currently working on a deployment script, to make my own workflow a little easier.  In doing so, I have it run the command `php app/nut database:check`, and I would like to see the alterations proposed when doing so, as you also can on the web interface.

I've already implemented this change on my fork, in [this commit](https://github.com/brewneaux/bolt/commit/cd844b5805fad07ee1a2c7bfa9d8d1668507dcda).

This change will add the following option:

-s, --show-changes

```
vagrant@scotchbox:/var/www$ php app/nut help database:check
Usage:
  database:check [options]

Options:
  -s, --show-changes    Show proposed schema changes 
  -h, --help            Display this help message
  -q, --quiet           Do not output any message
  -V, --version         Display this application version
      --ansi            Force ANSI output
      --no-ansi         Disable ANSI output
  -n, --no-interaction  Do not ask any interactive question
  -v|vv|vvv, --verbose  Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug

Help:
 Check the database for missing tables and/or columns.
```

**Comment by GawainLynch** [comment]
@brewneaux: This sounds like an awesome idea :+1: 

My only recommendation is to have a look at the Symfony Console TableHelper for output generation … it's super easy to use and is really nice output.

Also, if you PR, can you target master please. `release/3.0` is bug-fix only and we're about to branch `release/3.1` so this will target 3.2

**Comment by brewneaux** [comment]
@GawainLynch Thanks for the comments!

I opened a PR against master.

**Comment by GawainLynch** [comment]
Awesome mate, one nitpick comment added to said PR!

**Comment by SahAssar** [comment]
Closed/fixed in #5604
